agent/backend/backend.go

```diff
@@ -57,6 +58,7 @@ type Backend interface {
 	Start(ctx context.Context, cancelFunc context.CancelFunc) error
 	Stop(ctx context.Context) error
 	FullReset(ctx context.Context) error
+	ScrapeOpenTelemetry(ctx context.Context, policyName string) error
```

Comment:
I don't think it makes sense to for this to be part of the Backend interface. This should happen in the backend during ApplyPolicy, as we're doing now.

Reply:
fixed